### PR TITLE
visit close and check

### PR DESCRIPTION
**Pull request by IuliaDVSA**
VTA-300 automation

### CODE REVIEW

File: src/main/java/clients/ActivitiesClient.java [hunk 1]
```diff
@@ -284,4 +283,37 @@ public void insertActivityWithAlterations(String body, List<JsonPathAlteration>
     public void deleteActivity(String id) {
         AwsUtil.deleteActivityById(id);
     }
+
+    private Response callOpenVisitCheck(String activityType, String testerStaffId) {
```
Comment (cb-cs) [inline src/main/java/clients/ActivitiesClient.java:287]: Just a query, is the activityType value implemented across these methods ever used? 

Looking at the /activities/open endpoint, it only ever uses a testerStaffId in the lookup to dynamoDB.

File: src/main/java/steps/ActivitiesSteps.java [hunk 2]
```diff
@@ -30,6 +29,12 @@ public void postActivities(Activities activities) {
         response = activitiesClient.postActivities(activities);
     }
 
+    @Step
+    public Response postActivitiesGetResponse(Activities activities) {
+        response = activitiesClient.postActivities(activities);
```
Comment (cb-cs) [inline src/main/java/steps/ActivitiesSteps.java:34]: There is nothing wrong with this, but you can probably return in-line. 
ie - return activitiesClient.postActivities(activities);

File: src/main/java/clients/ActivitiesClient.java (hunk 1 above)
Comment (IuliaDVSA) [reply src/main/java/clients/ActivitiesClient.java]: Removed, as indeed not really necessary.

**Review by wis-dvsa — APPROVED** [review]

**Comment by IuliaDVSA** [comment]
VTA-300 / VTA-470  reviewed and approved

**Pull request merged**